packages/password-field/test/password-field.test.js

```diff
@@ -56,22 +64,19 @@ describe('password-field', () => {
   });
 
   it('should prevent touchend event on reveal button', () => {
-    const e = new CustomEvent('touchend', { cancelable: true });
-
-    revealButton.dispatchEvent(e);
-    expect(e.defaultPrevented).to.be.true;
+    const event1 = makeSoloTouchEvent('touchend', null, revealButton);
```

Comment:
As we use gesture events in `vaadin-button`, a manually created `CustomEvent` was causing errors.